tests/test_multiple_interpreters.py

```diff
@@ -86,7 +86,7 @@ def test_dependent_subinterpreters():
 
     sys.path.append(".")
 
-    if sys.version_info >= (3, 14):
+    if sys.version_info >= (3, 15):
```

Comment:
Is this final for 3.14, or could this still change before the 3.14.0 release?

If the latter: maybe add a comment?

Reply:
It will continue to work. I think `intepreters` is supposed to be released on PyPI during the 3.14 lifecycle.